go.mod

```diff
@@ -1,97 +1,101 @@
 module github.com/openshift/ocm-agent-operator
 
-go 1.19
+go 1.20
 
 require (
-	github.com/go-logr/logr v1.2.4
-	github.com/golang/mock v1.5.0
+	github.com/go-logr/logr v1.4.1
+	github.com/golang/mock v1.6.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/ginkgo/v2 v2.12.1
-	github.com/onsi/gomega v1.27.10
+	github.com/onsi/ginkgo/v2 v2.14.0
+	github.com/onsi/gomega v1.30.0
 	github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible
-	github.com/openshift/operator-custom-metrics v0.4.3-0.20220322205053-7b528cc0d6eb
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.55.0
-	github.com/prometheus/client_golang v1.15.1
+	github.com/openshift/operator-custom-metrics v0.5.1
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.71.2
+	github.com/prometheus/client_golang v1.18.0
 	github.com/sykesm/zap-logfmt v0.0.4
-	go.uber.org/zap v1.24.0
-	k8s.io/api v0.27.4
-	k8s.io/apimachinery v0.27.4
-	k8s.io/client-go v0.27.4
-	k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f
-	sigs.k8s.io/controller-runtime v0.15.0
-	sigs.k8s.io/controller-tools v0.11.3
+	go.uber.org/zap v1.26.0
+	k8s.io/api v0.29.1
+	k8s.io/apimachinery v0.29.1
+	k8s.io/client-go v0.29.1
```

Comment:
Can we stick to version 0.27.x (x being latest available in these deps) for api, apimachinery and client-go for Kube? That's the Kube version for OCP 4.14 which we want to stick with for now..

Reply:
Reference - https://docs.openshift.com/container-platform/4.14/updating/preparing_for_updates/updating-cluster-prepare.html#kube-api-removals_updating-cluster-prepare

Reply:
Sure, good spot. I'll make the update.